EventFilter/TotemRawToDigi/plugins/TotemTriggerRawToDigi.cc

```diff
@@ -0,0 +1,149 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com)
+*
+****************************************************************************/
+
+#include "FWCore/Framework/interface/one/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+#include "DataFormats/FEDRawData/interface/FEDRawData.h"
+#include "DataFormats/FEDRawData/interface/FEDRawDataCollection.h"
+#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
+
+#include "DataFormats/TotemDigi/interface/TotemTriggerCounters.h"
+
+#include <string>
+#include <iostream>
+
+//----------------------------------------------------------------------------------------------------
+
+class TotemTriggerRawToDigi : public edm::one::EDProducer<>
+{
+  public:
+    explicit TotemTriggerRawToDigi(const edm::ParameterSet&);
+    ~TotemTriggerRawToDigi();
+
+    virtual void produce(edm::Event&, const edm::EventSetup&) override;
+    virtual void endJob();
```

Comment:
I see that this method is empty.
The base class has it already, no need to overload ... add  ``` override ``` if kept